Change/vpn fixes ubuntu22

bartvanb opened the pull request:
(Partial?) fix for #724 

- building major+minor version support images
- no longer building support images twice each time
- use major+minor version support images that match node version by default

frankcorneliusmartin reviewed (changes requested):
Ok. Good that these images are now also versioned the samen. I am however a little bit worried that this does not fix the VPN issues as this was a problem with the 'newer' VPN images?

From the issue https://github.com/vantage6/vantage6/issues/724: 

> I replaced harbor2.vantage6.ai/infrastructure/vpn-configurator:latest with harbor2.vantage6.ai/infrastructure/vpn-configurator:3.10.3 then it worked again.

bartvanb commented:
> Ok. Good that these images are now also versioned the samen. I am however a little bit worried that this does not fix the VPN issues as this was a problem with the 'newer' VPN images?

> From the issue https://github.com/vantage6/vantage6/issues/724:

> > I replaced harbor2.vantage6.ai/infrastructure/vpn-configurator:latest with harbor2.vantage6.ai/infrastructure/vpn-configurator:3.10.3 then it worked again.

Honestly I am not sure and think it would be best to test a release candidate if all issues are solved. There was definitely an explanation for older version to work: at some point, we changed from the old version of `iptables` to the new version. At that point, people with the older version had to use the old image. Then, we changed again to support both the old and the new in the commit I mentioned. But I'm not 100% sure this is the case, but best to find out by testing IMO
